public/components/getting_started/components/utils.tsx

```diff
@@ -21,19 +20,23 @@ const fetchAssets = async (tutorialId: string, assetFilter?: 'dashboards' | 'ind
   return responeData;
 };
 
-export const UploadAssets = async (tutorialId: string) => {
+export const UploadAssets = async (tutorialId: string, mdsId: string, mdsLabel: string) => {
   const { setToast } = useToast();
-  try {
-    const responeData = await fetchAssets(tutorialId);
+  const http = coreRefs.http;
 
-    const blob = new Blob([responeData.data], { type: 'application/x-ndjson' });
-    const file = new File([blob], 'ndjson-file.ndjson');
+  try {
+    const response = await http!.post(`/api/observability/gettingStarted/mdsLoading`, {
+      body: JSON.stringify({
+        mdsId,
+        mdsLabel,
+        tutorialId,
+      }),
+    });
 
-    const error = await uploadBundle(file);
-    if (error) {
-      console.error(error.message);
-    } else {
+    if (response) {
       setToast('Created saved object assets successfully', 'success');
+    } else {
+      setToast('Failed to create saved object assets', 'danger');
```

Comment:
Will this lead to double toast on errors? we are already setting error toast in `catch` phrase below

Reply:
Yes, good catch has been removed.